src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/CommonPageElements.java

```diff
@@ -0,0 +1,195 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CommonPageElements extends BasePage {
+    ServiceDifferentElementPage serviceDifferentElementPage;
+
+    @FindBy(xpath = "//li[@class='dropdown uui-profile-menu']//span[@class='caret'][1]")
+    protected WebElement openLoginDropdownButton;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::input[@id='name']")
+    protected WebElement loginField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::input[@id='password']")
+    protected WebElement passwordField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[text()='Enter']")
+    protected WebElement enterCredentials;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[@id='user-name']")
+    WebElement loggedUserName;
+
+    @FindBy(xpath = "//header//ul/li/a[@href='index.html'][1]")
+    WebElement headerItemHome;
+
+    @FindBy(xpath = "//header//ul/li[2]/a[1]")
+    WebElement headerItemContact;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle'][1]")
+    WebElement headerItemService;
+
+    @FindBy(xpath = "//header//ul/li/a[@href='metals-colors.html'][1]")
+    WebElement headerItemMetalAndColor;
+
+    @FindBy(xpath = "//ul[@class='sidebar-menu left']")
+    WebElement leftSection;
+
+    @FindBy(xpath = "/html/body/footer")
+    WebElement pageFooter;
+
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/"
+            + "a[text()='Support']")
+    WebElement subMenuFromHeaderSupportItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Dates']")
+    WebElement subMenuFromHeaderDatesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Search']")
```

Comment:
`@FindBy(linkText = "")`?

Do you really need this element?

Reply:
yes, check header menu items